server/src/project/project.service.ts

```diff
@@ -283,8 +284,50 @@ export class ProjectService {
   constructor(
     private readonly dynamicsWebApi: OdataService,
     private readonly config: ConfigService,
+    private readonly carto: CartoService,
   ) {}
 
+  async getBblsGeoJSON(bbls) {
+    let collectedBBLs = bbls;
```

Comment:
I know this is code from old commits, but I would avoid re-naming the argument like this haha. Or just rename the arg. Idk.

Reply:
I had that thought too. But collectedBBLs is modified on the fly a few lines down. It's not so much a renaming as it is assigning a default value to collectedBBLs

Reply:
Though actually on second read it looks unnecessary. collectedBBs can just be declared on line 295 instead. 
Will change